src/compiler/scala/reflect/reify/package.scala

```diff
@@ -72,7 +72,7 @@ package object reify {
     if (tpe.isSpliceable) {
       val classTagInScope = typer0.resolveClassTag(enclosingMacroPosition, tpe, allowMaterialization = false)
       if (!classTagInScope.isEmpty) return Select(classTagInScope, nme.runtimeClass)
-      if (concrete) throw ReificationException(enclosingMacroPosition, "tpe %s is an unresolved spliceable type".format(tpe))
+      if (concrete) throw ReificationException(enclosingMacroPosition, "type %s is an unresolved spliceable type".format(tpe))
```

Comment:
The word is redundant and could be omitted. Also, could change `format` to `s"$tpe"`.

Reply:
> The word is redundant and could be omitted

Although technically redundant, maybe it reads a bit nicer this way? 🤷